libvast/src/detail/fill_status_map.cpp

```diff
@@ -21,7 +23,18 @@
 #include <caf/settings.hpp>
 #include <caf/stream_manager.hpp>
 
-#include "vast/detail/algorithms.hpp"
+#include <sstream>
+#include <thread>
+
+namespace {
+
+std::string thread_id() {
+  std::stringstream ss;
+  ss << std::this_thread::get_id();
+  return std::move(ss).str();
+}
```

Comment:
Isn't this just `std::to_string(thread_id)`?

Reply:
Nope, the only way to convert it is by going through `operator<<(std::ostream&, const std::thread::id&)`. See also: https://en.cppreference.com/w/cpp/thread/thread/id